libbeat/beat/event.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/elastic/beats/libbeat/common"
 )
 
+// FlagField fields used to keep information or errors when events are parsed.
+const FlagField = "log.flags"
```

Comment:
I'm wondering if dissect should write it's flags into `log.flags` or rather `event.flags`? Reasons is that dissect is not only for logs but more generic. 

Should have spotted this earlier.

Reply:
@webmat I think we need `event.flags` in the future in ECS.

Reply:
@ruflin Would it be the same for when an event is `truncated`?

Reply:
I did create https://github.com/elastic/ecs/issues/100 a while ago for the `log` tag field. There is no issue yet for a more generic set of flags.

I agree with @ruflin that the dissect error should not be set on `log.flags`.

`event.flags` is a bit better. But I think this approach still mixes up pipeline & processing metadata with userland data (like the `error` discussion we had last week, @ruflin). The following idea hasn't been fleshed out yet, but I've been thinking we should introduce a section that's clearly about stuff that happened in the processing pipeline. E.g. `pipeline.error`, `pipeline.tags` (or flags), if someone wants to note down timings of each step in their pipeline, they'd do it under `pipeline.` as well, etc. However this will have to come after ECS 1.0/GA, so don't wait on this being defined for what needs to happen in Beats.

In the meantime, what I would suggest instead is to do what we've been doing for years, and add this dissect tag to `tags` directly, like Logstash does with `_grok_parse_failure`.

Reply:
And @ph, to answer your more recent question, I would consider the truncation to be userland information, about the log itself. So I do think having `truncated` right on `log.flags` makes sense.

This is the new field where the `multiline` tag is being added, correct? (Sorry I haven't been following these developments very closely)

Reply:
I believe it's the same field correct.

Reply:
Ok thanks for confirming. So my opinion for now is that flags that are descriptive of the log itself or the log entry should be added to `log.flags`, so `multiline`, `truncated`, etc.

Parsing flags like `dissect_parsing_error`, on the other hand, should be added to `tags`, until we define a more general place to put pipeline errors and details.

Reply:
To not block this PR, lets go with `log.flags` for now. Lets open a more general discussion where information from processing should go.

For `tags` in LS: We should probably also tackle this.